ext/native/file/file_util.cpp

```diff
@@ -17,6 +17,8 @@
 #include <string>
 #include <set>
 #include <algorithm>
+#include <strings.h>
```

Comment:
The strings.h include should be there now (higher up, and not on Windows to avoid conflicts.)

-[Unknown]